kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -346,7 +414,14 @@ private String sanitizeTableName(String tableName) {
    *         name if sanitization is enabled.
    */
   public String getTableFromTopic(String topic) {
-    return getBoolean(SANITIZE_TOPICS_CONFIG) ? sanitizeTableName(topic) : topic;
+    String tableName = null;
+    Map<String, String> map = getTopicsToTables();
```

Comment:
I tried moving the method over but realized it's also called in the BigQuerySinkConfig (getTablesToTopics) and in the BigQuerySinkConnector (ensureExistingTables). Since we want to be able to access the map in all three classes, is there a way to store the map as a variable in the config class?